spec/topological_inventory/openshift/collector_spec.rb

```diff
@@ -5,6 +5,7 @@
       "mock_openshift.redhat.com",
       "8443",
       "secret",
+      metrics
```

Comment:
If we allow metrics to be nil we might not need to change this spec at all?

Reply:
Sure but then we would need to change all (just one currently) the callers to methods on that object to check for nil when that would only happen in test. Was trying to avoid that, but it looks like the port isn't getting released quickly enough for it to be created an brought down for each spec.